doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Linux/index.md

```diff
@@ -31,9 +31,9 @@ Before building the C++ endpoint SDK, install the following components on your m
 4. Install the [AvroC++](http://avro.apache.org/docs/1.7.6/api/cpp/html/index.html) library manually:
 
    ```
-   wget http://archive.apache.org/dist/avro/avro-1.7.5/cpp/avro-cpp-1.7.5.tar.gz
+   wget https://archive.apache.org/dist/avro/avro-1.7.7/cpp/avro-cpp-1.7.7.tar.gz
    tar -zxf avro-cpp-1.7.7.tar.gz
```

Comment:
Fix this line instead as we use avro-cpp-1.7.5 in all other guides, SDKs and server. Don't update version in the guide as it wasn't generally tested.